File: 2673/2673.cpp
class Solution {
public:
    int makeCostsOfPathsEqualInABinary(TreeNode* root) {
        if(!root) return 0;
        return 1+max(makeCostsOfPathsEqualInABinary(root->left),makeCostsOfPathsEqualInABinary(root->right));
    }
};

File: 2673/2673.cs
public class Solution {
    public int MakeCostsOfPathsEqualInABinary(TreeNode root) {
        if (root == null) return 0;
        return 1 + Math.Max(MakeCostsOfPathsEqualInABinary(root.left), MakeCostsOfPathsEqualInABinary(root.right));
    }
}